android/app/src/main/java/io/mosip/registration_client/api_services/SyncResponseApi.java

```diff
@@ -0,0 +1,676 @@
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import io.mosip.registration.clientmanager.BuildConfig;
+import io.mosip.registration.clientmanager.dto.CenterMachineDto;
+import io.mosip.registration.clientmanager.dto.http.CertificateResponse;
+import io.mosip.registration.clientmanager.dto.http.ClientSettingDto;
+import io.mosip.registration.clientmanager.dto.http.IdSchemaResponse;
+import io.mosip.registration.clientmanager.dto.http.MasterData;
+import io.mosip.registration.clientmanager.dto.http.ResponseWrapper;
+import io.mosip.registration.clientmanager.dto.http.ServiceError;
+import io.mosip.registration.clientmanager.dto.http.UserDetailResponse;
+import io.mosip.registration.clientmanager.entity.GlobalParam;
+import io.mosip.registration.clientmanager.entity.MachineMaster;
+import io.mosip.registration.clientmanager.entity.RegistrationCenter;
+import io.mosip.registration.clientmanager.entity.SyncJobDef;
+import io.mosip.registration.clientmanager.repository.ApplicantValidDocRepository;
+import io.mosip.registration.clientmanager.repository.BlocklistedWordRepository;
+import io.mosip.registration.clientmanager.repository.DocumentTypeRepository;
+import io.mosip.registration.clientmanager.repository.DynamicFieldRepository;
+import io.mosip.registration.clientmanager.repository.GlobalParamRepository;
+import io.mosip.registration.clientmanager.repository.IdentitySchemaRepository;
+import io.mosip.registration.clientmanager.repository.LanguageRepository;
+import io.mosip.registration.clientmanager.repository.LocationRepository;
+import io.mosip.registration.clientmanager.repository.MachineRepository;
+import io.mosip.registration.clientmanager.repository.RegistrationCenterRepository;
+import io.mosip.registration.clientmanager.repository.SyncJobDefRepository;
+import io.mosip.registration.clientmanager.repository.TemplateRepository;
+import io.mosip.registration.clientmanager.repository.UserDetailRepository;
+import io.mosip.registration.clientmanager.service.MasterDataServiceImpl;
+import io.mosip.registration.clientmanager.spi.JobManagerService;
+import io.mosip.registration.clientmanager.spi.SyncRestService;
+import io.mosip.registration.clientmanager.util.SyncRestUtil;
+import io.mosip.registration.keymanager.dto.CertificateRequestDto;
+import io.mosip.registration.keymanager.dto.CryptoRequestDto;
+import io.mosip.registration.keymanager.dto.CryptoResponseDto;
+import io.mosip.registration.keymanager.spi.CertificateManagerService;
+import io.mosip.registration.keymanager.spi.ClientCryptoManagerService;
+import io.mosip.registration.keymanager.util.CryptoUtil;
+import io.mosip.registration.packetmanager.util.JsonUtils;
+import io.mosip.registration_client.model.SyncPigeon;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+@Singleton
+public class SyncResponseApi implements SyncPigeon.SyncApi {
```

Comment:
Can we rename it as MasterDataSyncApi

Reply:
Yes, possible. I will change the class name to `MasterDataSyncApi`.